_posts/2021-03-29-Encoding the Lending Library.md

```diff
@@ -0,0 +1,214 @@
+					<note>returned</note>
+					<note>(Composition
+						<member-name>Gertrude Stein</member-name>
+						. gave copy to
+						<member-name>R W</member-name>)
+					</note>
+					<date-start>June 20 1946</date-start>
+					<title>The Negro in our History</title>
+					<title>Proletarian Literature</title>
+				</text>
+			</borrowing-record>
+		</side>
+		<side type="reverse"></side>
+	</div>
+	<div type="lending-library-card">
+		<side type="front">
+			<heading>
+				<text color="purple">
+					<member-name>Ellen Wright</member-name>
+				</text>
+				<text color="black">
+					<address>14 rue M. le Prince</address>
+				</text>
+			</heading>
+			<borrowing-record>
+				<text color="purple">
+					<date-start>Sept 19 1949</date-start>
+					<title>The Pasquier Chronicles</title>
+				</text>
+				<text color="blue">
+					<date-end>Oct 26</date-end>
+				</text>
+				<text color="purple">
+					<strikethrough>
+						<title>The Close</title>
+					</strikethrough>
+					<title>Christine</title>
+				</text>
+				<text color="blue">
+					<date-end>Oct 26 1949</date-end>
+				</text>
+				<text color="purple">
+					<title>Lafcadio's Adventures</title>
+				</text>
+				<text color="blue">
+					<date-end>Oct 26 1949</date-end>
+					<date-start>Oct 26 1949</date-start>
+					<title>The Gambler</title>
+					<date-start>Oct 26</date-start>
+					<title>Walter Rathenau his life &amp; work by
+						
+						<author>Count Harry Kessler</author>
+						<publication-info>
+							<publisher>Gerald Howe Ltd</publisher>
+							<address>23 Soho Square London</address>
+							<year>1929</year>
+						</publication-info>
+					</title>
+				</text>
+				<text color="faded-blue">
+					<note>
+						<unclear>Also wants	
+							<author>Christina</author>
+						</unclear>
+						&amp;	
+						<author>Morand's</author>
+						Books
+					</note>
+				</text>
+			</borrowing-record>
+		</side>
+		<side type="reverse"/>
+	</div>
+</div>
+```
+
+### Organizing Information: Categories and Hierarchy
+In order to categorize the information, I created a root tag that nested the separate cards. Within a div element for each card, I separated the encoding further with a ```<heading>``` element and a ```<borrowing-record>``` element. This first element contains Wright's name/address, while the second contains the remaining information on the card, including specific borrowed books and notes. 
+
+The first question I encountered was how to encode the dates listed on the cards. While I considered creating an element for each book which would include the borrowing date, data on the book itself, and the return date, many of the books were listed under one borrow date, and it did not feel accurate to repeatedly include the same date if it was not written on the original document. I also considered nesting the information about a book inside the tags related to its borrow date, but chose not to as borrow/return dates were not consistently available for all entries. 
```

Comment:
I see you thought about the comment I raised above! Definitely a tricky question to navigate when structuring the data here.